include/modem/lte_lc.h

```diff
@@ -1201,6 +1228,9 @@ struct lte_lc_evt {
 
 		/** Payload for event @ref LTE_LC_EVT_NEIGHBOR_CELL_MEAS. */
 		struct lte_lc_cells_info cells_info;
+
+		/** Payload for event @ref LTE_LC_EVT_RAI_UPDATE. */
+		struct lte_lc_rai rai;
```

Comment:
The name `lte_lc_rai` is a bit ambiguous, I think. The struct could be called `lte_lc_rai_cfg` and the member `rai_cfg`. This would also be aligned with other payload types.